proto/internal/uber/cadence/matching/v1/service.proto

```diff
@@ -83,6 +83,12 @@ service MatchingAPI {
 
   // GetTaskListsByDomain returns all tasklist for a given domain
   rpc GetTaskListsByDomain(GetTaskListsByDomainRequest) returns (GetTaskListsByDomainResponse);
+
+  // UpdateTaskListPartitionConfig is called to update the partition config of a task list
+  rpc UpdateTaskListPartitionConfig(UpdateTaskListPartitionConfigRequest) returns (UpdateTaskListPartitionConfigResponse);
+
+  // RefreshTaskListPartitionConfig is called to update the cache of partition config of a task list
+  rpc RefreshTaskListPartitionConfig(RefreshTaskListPartitionConfigRequest) returns (RefreshTaskListPartitionConfigResponse);
```

Comment:
Please elaborate more on what these APIs are supposed to do and who calls them when. Also update PR description so it's more explicit for the reviewers